pkgs/by-name/ch/chasemapper/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3,
+}:
+let
+  customPython = python3.withPackages (
+    ps: with ps; [
+      eccodes
+      cusfpredict
+      flask
+      flask-socketio
+      lxml
+      numpy
+      python-dateutil
+      pytz
+      requests
+      pyserial
+      simple-websocket
+    ]
+  );
+in
+stdenv.mkDerivation rec {
+  pname = "chasemapper";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "projecthorus";
+    repo = "chasemapper";
+    rev = "v${version}";
```

finalAttrs is generally preferred
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "chasemapper";
  version = "1.5.3";

  src = fetchFromGitHub {
    owner = "projecthorus";
    repo = "chasemapper";
    rev = "v${finalAttrs.version}";
```